src/dbnode/storage/cluster/database.go

```diff
@@ -409,13 +407,11 @@ func (d *clusterDB) analyzeAndReportShardStates() {
 		count := d.bootstrapCount[id]
 		if count != len(namespaces) {
 			// Should never happen if bootstrapped and durable.
```

Comment:
nit: Update this comment to say that this can temporarily occur due to race condition?